darkknight_gpg/apps.py

```diff
@@ -0,0 +1,30 @@
+import gnupg
+
+from django.apps import AppConfig
+from django.conf import settings
+
+from darkknight.signals import csr_created
+from .models import EncryptedPrivateKey
+
+
+class DarkKnightGpgConfig(AppConfig):
+    name = 'darkknight_gpg'
+
+    def ready(self):
+        csr_created.connect(gpg_encrypt)
+
+
+def gpg_encrypt(sender, key, **kwargs):
+    with open(settings.GPG_PUBLIC_KEY_PATH) as f:
+        key_data = f.read()
+
+    gpg = gnupg.GPG()
```

Comment:
Tested locally... This is actually messing with my GPG instance because it doesn't set a `homedir`. Moreover, this app could be running as a user that can't write to it's home directory. (Certain configuration for www-data are like this.)

I dunno how you want to proceed... Either you want to update the README to say that you need to set the environment variable GPGHOME. Or you give the ability to set the HOMEDIR in the `settings.py` file.